### DISCUSSION TITLE
Thanks !

### DISCUSSION THREAD

**Discussion by AugmentedRealityCat** [zero01101/openOutpaint#115] [General]
**Thanks** a lot to all of you who made this AMAZING extension for Automatic1111 possible.
@seijihariki , @Kalekki , @lifeh2o, @jasonmhead, and of course @zero01101 

I just love everything about this tool. My favourite function is painting a larger area than the resolution SD is actually rendering at - some kind of _invert-high-res-fix_ (a _low-res-hack_ then ?). I was convinced something like this was possible, but to actually use it, it's really something else WOW ! this is even more useful than I thought. And then you can go back and in-paint the part you want in focus. I also really appreciate the new extension edition with it own tab directly inside the WebUI. It's really convenient.

**Comment by zero01101** [comment]
![myfacewhen](https://user-images.githubusercontent.com/1649724/209158672-87f609ad-a6b2-43e2-aa1f-0fb4dd65f165.png)

aww, many thanks for the kind words <3 glad you're enjoying!

**Comment by seijihariki** [comment]
Happy people are liking using it! Please enjoy!
